examples/domain_gap/tests/test_tensorleap.py

```diff
@@ -0,0 +1,56 @@
+import matplotlib.pyplot as plt
+from tensorleap import *
+
+
+def iter_all_samples(data: List[PreprocessResponse]):
+    err_i = 0
+    for sub in data:
+        for i in range(sub.length):
+            try:
+                x = input_image(i, sub)
+                mask = ground_truth_mask(i, sub)
+                city_val = metadata_city(i, sub)
+                err_i = 0
+            except Exception as e:
+                print(f"Error in processing sample {i} from {sub['subset_name']} set")
+                err_i += 1
+                if err_i > 10:
```

Comment:
Why is this line here and we stop at 10? is the same thing happens on the engine? o.w. I would not continue the script here - but let the error propogate all the way through (either with print(e) or without the try - except)